packages/components/addon/components/hds/code-block/copy-button.hbs

```diff
@@ -0,0 +1,13 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Copy::Button
+  class="hds-code-block__copy-button"
+  @text="Copy"
+  @isIconOnly={{true}}
```

Comment:
Let's not forget that soon this button will become even smaller with https://github.com/hashicorp/design-system/pull/1690. Should we also add a visible "Copy" label, or it's enough to have the icon only?

Reply:
I don't think text is necessary for a couple of reasons:

1. The paradigm of this type of component is pretty well understood, without making an assumption, end-users probably have a bit of contextual knowledge just based on the structure of the component what this button does. I also think the icon is explicit enough.
2. It's possible that adding text may detract from the actual code, which should be the main focus point of the end-user.

If explicit is better, it's not hard to add, here are a [couple of examples](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/wMiLXorCLapCZoPGARoXHN/HDS-Product---Components?type=design&node-id=48551%3A5252&mode=design&t=ZMMDAfe3iP40ZOlS-1)

Reply:
Let's make sure if it's icon-only, text is still required for `aria-label`